addons/project/__manifest__.py

```diff
@@ -61,12 +61,12 @@
             'project/static/src/js/project_pivot.js',
             'project/static/src/js/project_rating_reporting.js',
             'project/static/src/js/project_task_kanban_examples.js',
-            'project/static/src/js/project_update_view.js',
-            'project/static/src/js/widgets/*',
-            'project/static/src/js/components/*',
             'project/static/src/js/tours/project.js',
             'project/static/src/js/project_calendar.js',
             'project/static/src/js/burndown_chart/*',
+            'project/static/src/js/right_panel/*',
+            'project/static/src/js/update/*',
+            'project/static/src/js/widgets/*',
```

Comment:
I think you should move the [MOV] commit for a widget could be after this commit since you define the path only from this commit.

Reply:
I'll rewrite this.